src/main/java/au/org/democracydevelopers/raireservice/response/GenerateAssertionsResponse.java

```diff
@@ -32,19 +31,16 @@ the raire assertion generation engine (https://github.com/DemocracyDevelopers/ra
  */
 public class GenerateAssertionsResponse {
 
-  protected final static Logger logger = LoggerFactory.getLogger(
-      au.org.democracydevelopers.raireservice.request.ContestRequest.class);
+  protected final static Logger logger = LoggerFactory.getLogger(GenerateAssertionsResponse.class);
```

Comment:
The other ones are private. I don't think it really matters, but might as well make them all private.